tensorbay/opendataset/BDD100K/loader.py

```diff
@@ -265,14 +265,15 @@ def _add_poly2d_label_100k(
         labeled_polygon = LabeledPolygon(
             points=poly2d_info["vertices"],
             category=label_info["category"],
-            attributes=label_info["attributes"],
         )
         polygon.append(labeled_polygon)
     else:
+        attributes = label_info["attributes"]
+        attributes.pop("laneTypes", None)
```

Comment:
if attributes will not be `{}`, why we use `pop` to delete item, can we use `del` instead?